client-server-client/src/main/scala/zio/raft/client/ClientConfig.scala

```diff
@@ -0,0 +1,44 @@
+package zio.raft.client
+
+import zio.*
+import zio.raft.protocol.MemberId
```

Comment:
why are coupling this to Raft? I thought the idea is to have a library that can be used for many products not just raft based ones